### DISCUSSION TITLE
What is the default for "Upscaler for img2img"?

### DISCUSSION THREAD

**Discussion by scpedicini** [vladmandic/sdnext#489] [Q&A] [answered]
I was looking through the Upscaling settings and noticed that the "Upscaler for img2img" is set to `SwinIR 4x`. It seems to use this upscaler for both img2img as well as inpainting (masked regions that are scaled up to add detail). I checked my automatic1111 install, and the default Upscaler is set to `None`.

Just wondering what the reason was for this change. I do a great deal of inpainting especially "Only Masked" so its making use of the upscaler quite a lot. I mostly work on fantasy digital illustrations (not photographic) - just looking for general advice if SwinIR is the best setting for this or if I should switch it to `None`.

![CleanShot 2023-04-25 at 10 02 20@2x](https://user-images.githubusercontent.com/2040540/234319402-86116ce9-a782-4ea1-b0a1-dcdb7e9a9aca.jpg)

**Comment by vladmandic** [comment]
default is just a default. i like it and other like others - feel free to pick whichever you want or add a different one. as long as it comes from ESRGAN/R-ESRGAN/SwinIR family, its just a question of downloading model and placing it in a folder.

**Reply by scpedicini** [reply]
@vladmandic yeah I get that. The reason I asked is that I've heard that the scalers labeled "4x" produce their best results at 4x multiplication, but obviously with the inpainting masked only, the scaling will be all kinds of ratios so was wondering if SwinIR 4x would be the most appropriate fit for something like this. 

Additionally, I've found purely anecdotally that SinIR has a bit of a "painter" feeling to it and it less on the photorealistic side, so it might not be the best fit when working on certain images.

Since this setting is sort of hidden (I doubt your average user even realizes that they can set a specific upscaler for their inpainting), it might be better to expose this as a dropdown choice in the InPainting tab so that they can switch it around as necessary.

**Reply by vladmandic** [reply, marked as answer]
yes, thats a valid suggestion - do you mind opening a feature request for it so it doesn't fall off the radar?

**Reply by scpedicini** [reply]
Done!

https://github.com/vladmandic/automatic/issues/546
